modules/atomic-widgets/assets/js/editor/div-block-view.js

```diff
@@ -193,26 +193,11 @@ const DivBlockView = BaseElementView.extend( {
 		};
 	},
 
-	getDroppableAxis() {
-		if ( this.isHorizontalAxis() ) {
-			return 'horizontal';
-		}
-
-		return 'vertical';
-	},
-
-	isHorizontalAxis() {
-		const styles = window.getComputedStyle( this.$el[ 0 ] );
-
-		return 'flex' === styles.display &&
-			[ 'row', 'row-reverse' ].includes( styles.flexDirection );
-	},
-
 	getDroppableOptions() {
 		const items = '> .elementor-element, > .elementor-empty-view .elementor-first-add';
 
 		return {
-			axis: this.getDroppableAxis(),
```

Comment:
The problem with the droppable axis is that they didn't take into consideration that inside the V4 containers we can update the flex direction styling at any point, without forcing a re-rendering of the container.